p3.py

```diff
@@ -1,3 +1,4 @@
+#start
 #https://leetcode.com/problems/roman-to-integer/
 #clink on above link to see problem definition 
 class Solution:
```

Comment:
1. Remove unnecessary integer conversions
2. Saves lines and makes the code neat when you initialize a good number of variables in a single line
For Example,
`r, j, sum, k = "IVXLCDM00", len(s), 0, 0`

Look at some solutions on leetcode. 
This one is good: [https://leetcode.com/problems/roman-to-integer/discuss/6547/Clean-O(n)-c%2B%2B-solution](url)